addon/helpers/fn-if.js

```diff
@@ -1,8 +1,8 @@
 import { helper } from '@ember/component/helper';
 
-export function fnIf([condition, invokeable]/*, hash*/) {
+export function fnIf([condition, invokeable, ...args]/*, hash*/) {
   if (condition) {
-    return invokeable;
+    return () => invokeable(...args);
```

Comment:
any arguments passed to this function should be appended to the end to match `fn` (and prior behavior)
```suggestion
    return (...extra) => invokeable(...args, ...extra);
```

Reply:
Sounds good.